src/Modules/UngradedModules/Presentation.jl

```diff
@@ -521,6 +521,12 @@ function prune_with_map(M::ModuleFP{T}) where {T<:MPolyRingElem{<:FieldElem}} #
   pm = presentation(M)
   pres_map = map(pm, 1)
   krnel = SubquoModule(pm[0], pres_map.(gens(pm[1]))) # Create the image of pres_map without the inclusion map
+
+  # work around if we're passing the zero module to singular
```

Comment:
Maybe add the link to the Singular issue in the comment for future reference.
```suggestion
  # work around if we're passing the zero module to singular, see
  # https://github.com/oscar-system/Singular.jl/issues/796
```